editor/plugins/script_editor_plugin.cpp

```diff
@@ -2059,6 +2059,9 @@ void ScriptEditor::_update_script_names() {
 		_update_members_overview();
 		_update_help_overview();
 	} else {
+		call_deferred("_update_members_overview");
+		call_deferred("_update_help_overview");
```

Comment:
Doesn't this defeat the purpose of the `waiting_update_names` check which seems to specifically aim at preventing updates at the wrong time?

Reply:
Not really. The purpose of that check is to avoid doing it during that frame. And the deferred call respects that.

Reply:
Probably best to work out why `_go_to_tab` is not updating these rather then calling the methods twice. Seems at least for help files, we switch to the tab before updating the contents so `_update_help_overview
` see's it as empty.

Reply:
@Paulb23 The only difference between `3.2` and the `master` branch is that `ScriptEditor::_tree_changed()` is getting called on the latter when it wasn't previously, which causes the bug. Something is triggering the `tree_changed` signal when it shouldn't.